idl/github.com/uber/cadence/shared.thrift

```diff
@@ -1251,6 +1251,7 @@ struct TaskListStatus {
   10: optional i64 (js.type = "Long") backlogCountHint
   20: optional i64 (js.type = "Long") readLevel
   30: optional i64 (js.type = "Long") ackLevel
+  35: optional double ratePerSecond
```

Comment:
I would also add it to the PollerInfo. It might help troubleshooting situation when different pollers ask for a different rate.

Reply:
done